.github/workflows/_reusable_build_package.yml

```diff
@@ -290,6 +290,16 @@ jobs:
           echo "package_path=${PACKAGE_PATH}" >> $GITHUB_OUTPUT
           echo "package_filename=$(basename ${PACKAGE_PATH})" >> $GITHUB_OUTPUT
 
+      - name: Sign MSI
+        uses: skymatic/code-sign-action@v3
+        with:
+          certificate: '${{ secrets.MICROSOFT_CERTIFICATE }}'
+          password: '${{ secrets.MICROSOFT_CERTIFICATE_PASSWORD }}'
+          certificatesha1: '${{ secrets.MICROSOFT_CERTHASH }}'
+          certificatename: '${{ secrets.MICROSOFT_CERTNAME }}'
+          description: '${{ secrets.MICROSOFT_DESCRIPTION }}'
+          folder: ./msi/wix/bin/${{ matrix.platform }}/en-US
```

Comment:
We should pass these secrets in as arguments to the workflow, and also handle the case where they're not set, same as for MacOS signing. Otherwise, workflows started by dependabot will fail.

Reply:
I've had to lodge a ticket with IT to get access to those secrets. In the meantime Raj has added them to this repo.

`secrets.MICROSOFT_CERTNAME` is not in the 1password vault, but I don't think it's needed as it wasn't defined on the other repo either. I'll remove it from the configuration.